python/dgl/frame.py

```diff
@@ -293,7 +293,8 @@ def to(self, device, **kwargs):  # pylint: disable=invalid-name
             A new column
         """
         col = self.clone()
-        col.device = (device, kwargs)
+        col.storage = F.copy_to(col.storage, device, **kwargs)
+        col.device = None
```

Comment:
it this correct?

Reply:
Should be correct. The device seems to be a lazy flag indicating the storage should be move a particular device before use.